packages/test-utils/src/matches.js

```diff
@@ -56,7 +56,12 @@ export function matches(node, selector) {
     return element && element.matches && element.matches(selector.value)
   }
 
-  const componentInstance = node[FUNCTIONAL_OPTIONS] || node.child
+  // ignore the functional component if its a RouterView
+  // Find a good explanation comment why tahts the case, please help!
+  const componentInstance =
+    node[FUNCTIONAL_OPTIONS]?.name === 'RouterView'
```

Comment:
Can we have an extra check? Just to be sure:

`node[FUNCTIONAL_OPTIONS] && node[FUNCTIONAL_OPTIONS].name`

I am not sure if we have `?` enabled in this code base or not.

Reply:
Edit: we do **not** - that's why CI is failing to build. Let's do it old school for now with the `&&` check.

Reply:
okok